examples/benchmark.py

```diff
@@ -131,7 +164,7 @@ def runOneTest(testName, options):
 parser = ArgumentParser()
 platformNames = [mm.Platform.getPlatform(i).getName() for i in range(mm.Platform.getNumPlatforms())]
 parser.add_argument('--platform', dest='platform', choices=platformNames, help='name of the platform to benchmark')
-parser.add_argument('--test', dest='test', choices=('gbsa', 'rf', 'pme', 'apoa1rf', 'apoa1pme', 'apoa1ljpme', 'amoebagk', 'amoebapme'), help='the test to perform: gbsa, rf, pme, apoa1rf, apoa1pme, apoa1ljpme, amoebagk, or amoebapme [default: all]')
+parser.add_argument('--test', dest='test', choices=('gbsa', 'rf', 'pme', 'apoa1rf', 'apoa1pme', 'apoa1ljpme', 'amoebagk', 'amoebapme', 'JAC',  'FactorIX', 'Cellulose', 'STMV'), help='the test to perform: gbsa, rf, pme, apoa1rf, apoa1pme, apoa1ljpme, amoebagk, or amoebapme [default: all]')
```

Comment:
Instead of just `JAC`, `FactorIX`, etc., would it make sense to call these `Amber20-JAC`, `Amber20-FactorIX` to emphasize they come from the hard-coded Amber20 benchmark suite URL?

Reply:
Good idea.  Let's also make them all lower case: `amber20-cellulose`, etc.  That's consistent with the other values, and also easier to type!  Also update the help text to list all values.

Reply:
Also, does 'JAC' mean 'DHFR'?  If so, let's call it that.

Reply:
@jchodera @peastman Sounds good, renamed to the amber20-cellulose style.  

JAC is DHFR; it has the same number of atoms as the "pme" test, but is consistently slightly faster.